public/javascripts/collections.js

```diff
@@ -1 +1,45 @@
 //add stuff here
+// const { getUserId } = require('../../auth')
+const baseRoute = 'http://localhost:8080'
```

Comment:
not using this variable

Reply:
https://github.com/sam-hearst/Comic-collection/pull/68/commits/646f30b1ff1f0345a272df0646b9e5b28adeee1f